swagger2/src/main/resources/com/cerner/beadledom/swagger2/ui.mustache

```diff
@@ -4,9 +4,9 @@
   <head>
     <meta charset="UTF-8">
     <title>Swagger UI</title>
-    <link rel="stylesheet" type="text/css" href="./swagger-ui.css" >
-    <link rel="icon" type="image/png" href="./favicon-32x32.png" sizes="32x32" />
-    <link rel="icon" type="image/png" href="./favicon-16x16.png" sizes="16x16" />
+    <link rel="stylesheet" type="text/css" href="//unpkg.com/swagger-ui-dist@3/swagger-ui.css" >
```

Comment:
Nice. Where did you learn about this CDN for the swagger assets?

Reply:
The swagger installation readme had a reference for it, its pretty neat: https://github.com/swagger-api/swagger-ui/blob/master/docs/usage/installation.md#unpkg

Reply:
Also, if I'm reading it right the `@3` will update the minor version every time a new one comes out and I believe you can change it for increased granularity if you wanted to specify a minor version as well.